Makefile

```diff
@@ -41,6 +41,10 @@ endif
 
 all: release-all
 
+depends:
+	cd contrib/depends && $(MAKE) HOST=$(target) && cd ../.. && mkdir -p build/$(target)/release
```

Comment:
HOST=$(target) ? That seems like a possible mixup.

Reply:
It's correct :)
Target is specified on the command line `make depends target=...`, HOST is the... target that will run the binaries..

Reply:
OK, thanks for confirming.